### PR TITLE
Track and log total distance walked

### PR DESCRIPTION
**Pull request by BrianDawson**
Short Description:  Tracks and logs total distance walked. Easy way to monitor how far you have walked, that way you know about when you can hatch your eggs.

### CODE REVIEW

File: pokemongo_bot/stepper.py [hunk 1]
```diff
@@ -31,6 +30,7 @@ def __init__(self, bot):
         self.steplimit2 = self.steplimit**2
         self.origin_lat = self.bot.position[0]
         self.origin_lon = self.bot.position[1]
+        self.totalDistance = 0
```
Comment (MaxLeiter) [inline pokemongo_bot/stepper.py]: this project generally uses _ instead of camelCasing

File: pokemongo_bot/stepper.py [hunk 2]
```diff
@@ -90,6 +91,7 @@ def _walk_to(self, speed, lat, lng, alt):
             self.api.set_position(lat, lng, alt)
             self.bot.heartbeat()
             logger.log("[#] Finished walking")
+            logger.log('[#] Total distance walked: ' + str(convert(self.totalDistance, "m", "km")) + ' kilometers | ' + str(convert(self.totalDistance, "m", "mi")) + ' miles.')
```
Comment (MaxLeiter) [inline pokemongo_bot/stepper.py]: Please have the unit displayed be the unit set in the config
https://github.com/PokemonGoF/PokemonGo-Bot/blob/master/config.json.example#L14

**Comment by bstpierr** [comment]
This won't necessarily be correct. The game will discount any movement that is done too quickly. I already have a PR to track distance as well as many other stats: https://github.com/PokemonGoF/PokemonGo-Bot/pull/1059

**Comment by BrianDawson** [comment]
@MaxLeiter Updated

**Comment by solderzzc** [comment]
we don't accept PR on master branch to keep thing easy.

**Pull request closed without merging**